### PR TITLE
Documenting hand-install of essentials

### PR DESCRIPTION
**Pull request by maslade**
Issue: #13495 

## What I did
Moved a section of docs to the top of the page with a different heading.

## How to test
N/A

### CODE REVIEW

File: docs/essentials/introduction.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ A major strength of Storybook are [addons](/addons/) that extend Storybook’s U
 - [Backgrounds](./backgrounds.md)
 - [Toolbars & globals](./toolbars-and-globals.md)
 
+### Installation
```
Comment (shilman) [inline docs/essentials/introduction.md:14]: Probably should note that this comes pre-installed with `sb init`.

**Review by shilman — APPROVED** [review]
LGTM, I'll let @jonniebigodes make the call on this.

File: docs/essentials/introduction.md (hunk 1 above)
Comment (jonniebigodes) [reply docs/essentials/introduction.md:14]: @maslade can you address the feedback that @shilman mentioned? Once that's addressed i think that we'll be good to go. 

Let me know once you've taken care of it.

Comment (maslade) [reply docs/essentials/introduction.md]: You bet - updated!

File: docs/essentials/introduction.md [hunk 2]
```diff
@@ -11,6 +11,30 @@ A major strength of Storybook are [addons](/addons/) that extend Storybook’s U
 - [Backgrounds](./backgrounds.md)
 - [Toolbars & globals](./toolbars-and-globals.md)
 
+### Installing
+If you created your project with `sb init` since Storybook 6, this is already installed for you and you can skip the rest of this section.
+
```
Comment (jonniebigodes) [inline docs/essentials/introduction.md:16]: @maslade we could a little bit better regarding the wording as it reads a bit off. We could probably go with something like:

```md
### installation

If you're running `sb init` to add Storybook to your project, the essentials package (`@storybook/addon-essentials`) is already installed and configured for you . You can skip the rest of this section.

If you're upgrading from a previous Storybook version, you'll need to run the following command in your terminal:
....
```

Let me know what do you think and we'll move from there.

**Review by jonniebigodes — CHANGES_REQUESTED** [review]
@maslade just one small tweak to the wording and we're good to go

File: docs/essentials/introduction.md (hunk 2 above)
Comment (maslade) [reply docs/essentials/introduction.md:16]: That's much better -- appreciate the suggestion. I've updated the PR.

Comment (jonniebigodes) [reply docs/essentials/introduction.md:16]: @maslade something is off. I'm still seeing the same content as initial. I've checked the commits and there's only one and that one's content is before the suggestion i made in the comment above? Could you check on your end and see what's wrong?

Comment (maslade) [reply docs/essentials/introduction.md]: Bad push -- please check again.

**Pull request merged**